src/sonic-py-common/sonic_py_common/device_info.py

```diff
@@ -374,6 +374,22 @@ def is_multi_npu():
     return (num_npus > 1)
 
 
+def is_voq_supervisor():
+    asic_conf_file_path = get_asic_conf_file_path()
```

Comment:
I feel it is better to use the platform_env.conf file (eg: https://github.com/Azure/sonic-buildimage/blob/master/device/arista/x86_64-arista_7800_sup/platform_env.conf) for defining voq_supervisor instead of using asic.conf file. Need to add API's for getting this file similar to get_asic_conf_file_path().

Reply:
Fixed. The supervisor card type determination is done using a variable "supervisor" form platform_env.conf instead of using asic.conf